### PR TITLE
Update changelog and version pin-ing

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
     ]}
 
 setup(name='readinglist',
-      version='1.1.dev0',
+      version='1.2',
```
Comment (leplatrem) [inline setup.py]: why not 1.1 ?

File: setup.py [hunk 2]
```diff
@@ -11,11 +11,11 @@
...
-    'colander',
-    'cornice',
-    'six',
-    'waitress',
-    'cliquet[postgresql]'
+    'colander==1.0',
+    'cornice==0.20.0',
+    'six==1.9.0',
+    'waitress==0.8.9',
+    'cliquet[postgresql]==1.2'
```
Comment (leplatrem) [inline setup.py]: should we mention 1.2.0 since we plan to upgrade them simultaneously ?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,10 +4,47 @@ Changelog
...
+* Every setting can be specified via an environment variable
+  (e.g. ``cliquet.storage_url`` with ``CLIQUET_STORAGE_URL``)
+* Logging now relies on `structlog <http://structlog.org>`_ (#78)
+* Logging output can be configured to stream JSON (#78)
+* New cache backend for PostgreSQL (#44)
+* Documentation was improved on various aspects (#64, #86)
+* Handle every backend errors and return 503 errors (#21)
+* State verification for OAuth2 dance now expires after 1 hour (#83)
+* Add the preview field for an article (#156)
+* Setup the readinglist scope (#16)
```
Comment (leplatrem) [inline CHANGELOG.rst]: OAuth scope

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,10 +4,47 @@ Changelog
...
+**New features**
+
+* Every setting can be specified via an environment variable
+  (e.g. ``cliquet.storage_url`` with ``CLIQUET_STORAGE_URL``)
+* Logging now relies on `structlog <http://structlog.org>`_ (#78)
+* Logging output can be configured to stream JSON (#78)
+* New cache backend for PostgreSQL (#44)
+* Documentation was improved on various aspects (#64, #86)
+* Handle every backend errors and return 503 errors (#21)
+* State verification for OAuth2 dance now expires after 1 hour (#83)
```
Comment (leplatrem) [inline CHANGELOG.rst]: issues/PR numbers won't match this repo. should we have a distinct paragraph for cliquet related stuff ?

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,10 +4,47 @@ Changelog
...
+* Prevent error when pagination token has bad format (#72)
+* List of CORS exposed headers were fixed in POST on collection (#54)
+
+**Internal changes**
+
+* Added a method in `cliquet.resource.Resource` to override known fields
+  (*required by Kinto*)
+* Every setting has a default value
+* Every end-point requires authentication by default
+* Session backend was renamed to cache (#96)
```
Comment (leplatrem) [inline CHANGELOG.rst]: I wouldn't mention cliquet internal changes here

File: CHANGELOG.rst (hunk 4 above)
Comment (Natim) [inline CHANGELOG.rst]: I can prefix them with ``mozilla-services/cliquet`` I though about splitting in two but it doesn't make much sense.

File: README.rst [hunk 6]
```diff
@@ -201,7 +201,7 @@ Revoke superuser privilege:
 :note:
 
     Alternatively the SQL initialization files can be found in the
-    *cliquet* source code (`cliquet/session/postgresql/schemal.sql` and
+    *cliquet* source code (`cliquet/cache/postgresql/schemal.sql` and
```
Comment (pdehaan) [inline README.rst]: Not sure if you want double backticks (``) around those strings.

File: CHANGELOG.rst [hunk 7]
```diff
@@ -4,10 +4,39 @@ Changelog
...
+------------------
+
+**Breaking changes**
+
+* `cliquet.storage.postgresql` now uses UUID as record primary key (mozilla-services/cliquet#70)
+* Settings ``cliquet.session_backend`` and ``cliquet.session_url`` were
+  renamed ``cliquet.cache_backend`` and ``cliquet.cache_url`` respectively.
+* FxA user ids are not hashed anymore (mozilla-services/cliquet#82)
+* Setting ``cliquet.retry_after`` was renamed ``cliquet.retry_after_seconds``
+* OAuth2 redirect url now requires to be listed in
```
Comment (ametaireau) [inline CHANGELOG.rst]: Also, OAuth relier had been disabled by default for readinglist.

**Comment by leplatrem** [comment]
Do not forget CLIQUET_CACHE_URL in docker compose

**Comment by leplatrem** [comment]
And session_* in readinglist.ini

File: loadtests/server.ini [hunk 8]
```diff
@@ -5,7 +5,7 @@ cliquet.project_name = readinglist
 cliquet.project_docs = https://readinglist.rtfd.org/
 cliquet.http_scheme = http
 cliquet.basic_auth_enabled = true
-cliquet.session_backend = cliquet.session.redis
+cliquet.cache_backend = cliquet.session.redis
```
Comment (leplatrem) [inline loadtests/server.ini]: cliquet.cache.redis

**Comment by pdehaan** [comment]
Probably not helpful, but it should be fairly trivial to auto-generate a CHANGELOG file using Git commits.
We use these [Git commit guidelines](https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines) which allows us to generate some decent looking [/CHANGELOG.md](https://github.com/mozilla/fxa-content-server/blob/master/CHANGELOG.md) files. The tool we use to generate the changelog uses Node+Grunt, but I'm sure it's just a few Git commands and loops under the hood.

File: docker-compose.yml [hunk 9]
```diff
@@ -10,5 +10,5 @@ web:
   ports:
    - "8888:8888"
```
Comment (leplatrem) [inline docker-compose.yml]: This is 8000:8000 for RL :/ #myfault

**Pull request merged**